blueprints/app/files/package.json

```diff
@@ -36,8 +36,40 @@
     "@embroider/webpack": "^2.1.1<% } %>",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2<% if (typescript) { %>",
-    "@typescript-eslint/eslint-plugin": "^5.57.1",
-    "@typescript-eslint/parser": "^5.57.1<% } %>",
+    "@glint/core": "^1.0.2",
```

Comment:
If we aren't setting up Glint for use in the actual app, then `@glint/core` is unneeded. That package doesn't have any types in it; it's just the CLI and language server machinery.